com.chartboost.mediation/Runtime/ChartboostMediationSettings.cs

```diff
@@ -30,7 +30,10 @@ public enum ChartboostMediationPartners
         [Description("tapjoy")] TapJoy = 2048,
         [Description("unity")] UnityAds = 4096,
         [Description("vungle")] Vungle = 8192,
-        [Description("yahoo")] Yahoo = 16384
+        [Description("yahoo")] Yahoo = 16384,
+        [Description("mobilefuse")] MobileFuse = 32768,
+        [Description("verve")] Verve = 65536,
+        [Description("hyprmx")] HyprMX = 131072
```

Comment:
If we keep going with 2^n we can only add 14 more partners and then we'll run out of int's. That might be soon...